Source/Scene/GlobeSurfaceTileProvider.js

```diff
@@ -907,7 +962,13 @@ define([
 
             ++tileProvider._usedDrawCommands;
 
-            command.debugShowBoundingVolume = (tile === tileProvider._debug.boundingSphereTile);
+            if (tile === tileProvider._debug.boundingSphereTile) {
+                if (defined(surfaceTile.orientedBoundingBox)) {
+                    createDebugOrientedBoundingBox(surfaceTile.orientedBoundingBox, Color.RED).update(context, frameState, commandList);
```

Comment:
Take a look at the large block of added code above - shouldn't that handle it? It no longer always creates everything - it only does so if the selected tile changes.